llvm/test/CodeGen/AArch64/sme2-mops.ll

```diff
@@ -0,0 +1,552 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
```

Comment:
Please rename this file, because `mops` has an overloaded meaning. What about `streaming-compatible-memory-ops.ll` ?

Reply:
Done.